### DISCUSSION TITLE
Weather widget fix will be available soon

### DISCUSSION THREAD

**Discussion by creewick** [creewick/MontereyRainmeter#62] [Announcements]
The development of .NET version is progressing slowly, but the current version also needs some attention.
A lot of users are reporting that the Weather widget is not working property.
That's why I'm working on a patch that will change its data provider from **OpenWeatherAPI** to **OpenMeteoAPI**.

Upcoming patch should fix such bug requests as:
- Unable to change city in widget
- No weather data is displayed
- API key is expired / not valid

Also, there will be no need to create an account and generate an API key, the weather widget will work straight **out of the box**.
Same API will be used in .NET version.

The patch will be available next **Tuesday, 9 May.** Stay tuned!

**Comment by jaccydee** [comment]
I just tried to install the Monterey_1.0.2.rmskin file and the weather widget didn't work. 
I uninstalled and then tried the zip file/manual installation method (MontereyRainmeter-1.0.2.zip) - and the weather widget worked with no issues.
